### DISCUSSION TITLE
How I can have logged in only 1 user with same account

### DISCUSSION THREAD

**Discussion by hovorma** [symfony/symfony#47278] [Q&A] [answered]
Hi all. In my project, I have restrictions that with the same account at the same time can be logged in only 1 user. Does Symfony have something by default for it or it should be all done in a custom way? I couldn't find it in the documentation. Can somebody help me?

**Comment by MatTheCat** [comment, marked as answer]
AFAIK Symfony does not offer such feature, but you should be able to implement it using [locks](https://symfony.com/doc/current/components/lock.html).

**Reply by hovorma** [reply]
@MatTheCat thanks a lot
